app/controllers/v0/education_career_counseling_claims_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module V0
+  class EducationCareerCounselingClaimsController < ClaimsBaseController
+    def create
```

Comment:
Do you have a spec for this new controller method?

Reply:
@LindseySaari I have added some specs. Thank you for bringing this up 🤘🏻